tox.ini

```diff
@@ -200,20 +200,10 @@ commands =
   codespell
 
 [testenv:lint]
-basepython: python3.10
```

Comment:
Contributors shouldn't require 3.10 to run the lint. The CI version will not matter as long as this works for any supported version. I verified the changes in this PR work for 3.8-3.11